strings/charoccurrence.go

```diff
@@ -0,0 +1,18 @@
+// charoccurrence.go
+// description: Chars occurrence in the sentence
+// details: An algorithm to count chars in string
+// author(s) [Moein](https://github.com/mo1ein)
+// see charoccurrence_test.go
+
+package strings
+
+func Count(text string) map[rune]int {
```

Comment:
Now this probably needs more context
```suggestion
func CountChars(text string) map[rune]int {
```

Reply:
- [x] fixed